### PR TITLE
Add v9.0.0 changelog entry

### PR DESCRIPTION
**Pull request by rekmarks**
The v9 changelog update. Contains some extra formatting changes to the file because I decided to run `prettier` on it to get rid of syntax warnings in my editor.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,16 +2,37 @@
...
+- [#10069](https://github.com/MetaMask/metamask-extension/pull/10069): Fetch swap quote refresh time from API
+- [#10045](https://github.com/MetaMask/metamask-extension/pull/10045): Fix token validation in Send flow
+- [#10048](https://github.com/MetaMask/metamask-extension/pull/10048): Fix display of boolean values when signing typed data
+- [#9992](https://github.com/MetaMask/metamask-extension/pull/9992): Improve Ethereum transaction params validation
+- [#9905](https://github.com/MetaMask/metamask-extension/pull/9905): Redesign view quote screens
+- [#9320](https://github.com/MetaMask/metamask-extension/pull/9320): Prevent hidden tokens from reappearing
+- [#10000](https://github.com/MetaMask/metamask-extension/pull/10000): Use consistent font size for modal top right Close links
+- [#10046](https://github.com/MetaMask/metamask-extension/pull/10046): Improve home screen notification appearance
+- [#10117](https://github.com/MetaMask/metamask-extension/pull/10117): Fix network settings Kovan block explorer link
+- [#10040](https://github.com/MetaMask/metamask-extension/pull/10040): Disable console in contentscript to reduce noise
```
Comment (Gudahtt) [inline CHANGELOG.md:23]: This commit was included in v8.1.9; I'll move it there.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,16 +2,37 @@
...
+- [#10070](https://github.com/MetaMask/metamask-extension/pull/10070): Enable eth_getProof RPC method
```
Comment (Gudahtt) [inline CHANGELOG.md:24]: This was included in v8.1.9 already, so I will remove it.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,16 +2,37 @@
...
+- [#9156](https://github.com/MetaMask/metamask-extension/pull/9156): Remove window.web3 injection
+- [#10039](https://github.com/MetaMask/metamask-extension/pull/10039): Add web3 shim usage notification (#10039)
+- [#8640](https://github.com/MetaMask/metamask-extension/pull/8640): Implement breaking window.ethereum API changes
+- [#8629](https://github.com/MetaMask/metamask-extension/pull/8629): Fix `eth_chainId` return values for Infura networks
+- [#10019](https://github.com/MetaMask/metamask-extension/pull/10019): Increase Chrome minimum version to v63
+- [#10135](https://github.com/MetaMask/metamask-extension/pull/10135): Fix error where a swap only completed the token approval transaction
+- [#10100](https://github.com/MetaMask/metamask-extension/pull/10100): Remove unnecessary swaps footer space when in dropdown mode
+- [#10069](https://github.com/MetaMask/metamask-extension/pull/10069): Fetch swap quote refresh time from API
+- [#10045](https://github.com/MetaMask/metamask-extension/pull/10045): Fix token validation in Send flow
+- [#10048](https://github.com/MetaMask/metamask-extension/pull/10048): Fix display of boolean values when signing typed data
```
Comment (Gudahtt) [inline CHANGELOG.md:16]: This was included in v8.1.9, so I'll remove it here.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,16 +2,37 @@
...
+
+- [#9156](https://github.com/MetaMask/metamask-extension/pull/9156): Remove window.web3 injection
+- [#10039](https://github.com/MetaMask/metamask-extension/pull/10039): Add web3 shim usage notification (#10039)
+- [#8640](https://github.com/MetaMask/metamask-extension/pull/8640): Implement breaking window.ethereum API changes
+- [#8629](https://github.com/MetaMask/metamask-extension/pull/8629): Fix `eth_chainId` return values for Infura networks
+- [#10019](https://github.com/MetaMask/metamask-extension/pull/10019): Increase Chrome minimum version to v63
+- [#10135](https://github.com/MetaMask/metamask-extension/pull/10135): Fix error where a swap only completed the token approval transaction
+- [#10100](https://github.com/MetaMask/metamask-extension/pull/10100): Remove unnecessary swaps footer space when in dropdown mode
+- [#10069](https://github.com/MetaMask/metamask-extension/pull/10069): Fetch swap quote refresh time from API
+- [#10045](https://github.com/MetaMask/metamask-extension/pull/10045): Fix token validation in Send flow
```
Comment (Gudahtt) [inline CHANGELOG.md:15]: This was included in v8.1.9, so I'll remove it here.

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,16 +2,37 @@
 
 ## Current Develop Branch
 
-## 9.0.0 Fri Dec 11 2020
+## 9.0.0 Fri Jan 8 2021
+
+- [#9156](https://github.com/MetaMask/metamask-extension/pull/9156): Remove window.web3 injection
+- [#10039](https://github.com/MetaMask/metamask-extension/pull/10039): Add web3 shim usage notification (#10039)
```
Comment (Gudahtt) [inline CHANGELOG.md:8]: Nit: the PR number here is redundant

**Review by Gudahtt — COMMENTED** [review]
I've just updated this to include the two most recent commits, and I've addressed some fairly straightforward feedback from my review.

**Review by Gudahtt — APPROVED** [review]
This all looks good to me now! Please take a look at my changes before you merge this @rekmarks to make sure they all look OK

**Review by Gudahtt — APPROVED** [review]

**Review by tmashuang — APPROVED** [review]

**Pull request merged**